[2025-03-30T11:10:25.530+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: inout_data_ingestion.ingest_dummy_data scheduled__2024-10-02T01:30:00+00:00 [queued]>
[2025-03-30T11:10:25.541+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: inout_data_ingestion.ingest_dummy_data scheduled__2024-10-02T01:30:00+00:00 [queued]>
[2025-03-30T11:10:25.543+0000] {taskinstance.py:2193} INFO - Starting attempt 1 of 2
[2025-03-30T11:10:25.589+0000] {taskinstance.py:2214} INFO - Executing <Task(PythonOperator): ingest_dummy_data> on 2024-10-02 01:30:00+00:00
[2025-03-30T11:10:25.604+0000] {standard_task_runner.py:60} INFO - Started process 697 to run task
[2025-03-30T11:10:25.613+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'inout_data_ingestion', 'ingest_dummy_data', 'scheduled__2024-10-02T01:30:00+00:00', '--job-id', '321', '--raw', '--subdir', 'DAGS_FOLDER/inout_data.py', '--cfg-path', '/tmp/tmpk2u9t582']
[2025-03-30T11:10:25.617+0000] {standard_task_runner.py:88} INFO - Job 321: Subtask ingest_dummy_data
[2025-03-30T11:10:25.654+0000] {warnings.py:109} WARNING - /home/***/.local/lib/python3.8/site-packages/***/settings.py:194: DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
  SQL_ALCHEMY_CONN = conf.get("database", "SQL_ALCHEMY_CONN")

[2025-03-30T11:10:25.698+0000] {task_command.py:423} INFO - Running <TaskInstance: inout_data_ingestion.ingest_dummy_data scheduled__2024-10-02T01:30:00+00:00 [running]> on host ff9d64b065e6
[2025-03-30T11:10:25.806+0000] {taskinstance.py:2510} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='inout_data_ingestion' AIRFLOW_CTX_TASK_ID='ingest_dummy_data' AIRFLOW_CTX_EXECUTION_DATE='2024-10-02T01:30:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-10-02T01:30:00+00:00'
[2025-03-30T11:10:25.818+0000] {inout_data.py:42} INFO - Successfully connected to MariaDB: INOUT on mariadb:3306
[2025-03-30T11:10:25.824+0000] {inout_data.py:63} ERROR - Error executing query: (1062, "Duplicate entry '2025-03-30 11:10:25-8842' for key 'PRIMARY'")
[2025-03-30T11:10:25.825+0000] {inout_data.py:63} ERROR - Error executing query: (1062, "Duplicate entry '2025-03-30 11:10:25-8842' for key 'PRIMARY'")
[2025-03-30T11:10:25.826+0000] {inout_data.py:63} ERROR - Error executing query: (1062, "Duplicate entry '2025-03-30 11:10:25-8842' for key 'PRIMARY'")
[2025-03-30T11:10:25.827+0000] {inout_data.py:63} ERROR - Error executing query: (1062, "Duplicate entry '2025-03-30 11:10:25-8842' for key 'PRIMARY'")
[2025-03-30T11:10:25.829+0000] {inout_data.py:63} ERROR - Error executing query: (1062, "Duplicate entry '2025-03-30 11:10:25-8842' for key 'PRIMARY'")
[2025-03-30T11:10:25.830+0000] {inout_data.py:63} ERROR - Error executing query: (1062, "Duplicate entry '2025-03-30 11:10:25-8842' for key 'PRIMARY'")
[2025-03-30T11:10:25.832+0000] {inout_data.py:63} ERROR - Error executing query: (1062, "Duplicate entry '2025-03-30 11:10:25-8842' for key 'PRIMARY'")
[2025-03-30T11:10:25.834+0000] {inout_data.py:63} ERROR - Error executing query: (1062, "Duplicate entry '2025-03-30 11:10:25-8842' for key 'PRIMARY'")
[2025-03-30T11:10:25.835+0000] {inout_data.py:159} INFO - Ingested 9 rows of dummy data.
[2025-03-30T11:10:25.836+0000] {inout_data.py:78} INFO - MariaDB connection closed.
[2025-03-30T11:10:25.838+0000] {python.py:202} INFO - Done. Returned value was: None
[2025-03-30T11:10:25.851+0000] {taskinstance.py:1149} INFO - Marking task as SUCCESS. dag_id=inout_data_ingestion, task_id=ingest_dummy_data, execution_date=20241002T013000, start_date=20250330T111025, end_date=20250330T111025
[2025-03-30T11:10:25.882+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2025-03-30T11:10:25.902+0000] {taskinstance.py:3309} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-03-30T11:17:01.214+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: inout_data_ingestion.ingest_dummy_data scheduled__2024-10-02T01:30:00+00:00 [queued]>
[2025-03-30T11:17:01.223+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: inout_data_ingestion.ingest_dummy_data scheduled__2024-10-02T01:30:00+00:00 [queued]>
[2025-03-30T11:17:01.224+0000] {taskinstance.py:2193} INFO - Starting attempt 1 of 2
[2025-03-30T11:17:01.234+0000] {taskinstance.py:2214} INFO - Executing <Task(PythonOperator): ingest_dummy_data> on 2024-10-02 01:30:00+00:00
[2025-03-30T11:17:01.238+0000] {standard_task_runner.py:60} INFO - Started process 2543 to run task
[2025-03-30T11:17:01.241+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'inout_data_ingestion', 'ingest_dummy_data', 'scheduled__2024-10-02T01:30:00+00:00', '--job-id', '1247', '--raw', '--subdir', 'DAGS_FOLDER/inout_data.py', '--cfg-path', '/tmp/tmpz4edeh_s']
[2025-03-30T11:17:01.242+0000] {standard_task_runner.py:88} INFO - Job 1247: Subtask ingest_dummy_data
[2025-03-30T11:17:01.253+0000] {warnings.py:109} WARNING - /home/***/.local/lib/python3.8/site-packages/***/settings.py:194: DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
  SQL_ALCHEMY_CONN = conf.get("database", "SQL_ALCHEMY_CONN")

[2025-03-30T11:17:01.273+0000] {task_command.py:423} INFO - Running <TaskInstance: inout_data_ingestion.ingest_dummy_data scheduled__2024-10-02T01:30:00+00:00 [running]> on host ff9d64b065e6
[2025-03-30T11:17:01.320+0000] {taskinstance.py:2510} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='inout_data_ingestion' AIRFLOW_CTX_TASK_ID='ingest_dummy_data' AIRFLOW_CTX_EXECUTION_DATE='2024-10-02T01:30:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-10-02T01:30:00+00:00'
[2025-03-30T11:17:01.323+0000] {inout_data.py:42} INFO - Successfully connected to MariaDB: INOUT on mariadb:3306
[2025-03-30T11:17:01.323+0000] {inout_data.py:63} ERROR - Error executing query: (1062, "Duplicate entry '2025-03-30 11:17:01-9176' for key 'PRIMARY'")
[2025-03-30T11:17:01.324+0000] {inout_data.py:63} ERROR - Error executing query: (1062, "Duplicate entry '2025-03-30 11:17:01-9176' for key 'PRIMARY'")
[2025-03-30T11:17:01.324+0000] {inout_data.py:63} ERROR - Error executing query: (1062, "Duplicate entry '2025-03-30 11:17:01-9176' for key 'PRIMARY'")
[2025-03-30T11:17:01.324+0000] {inout_data.py:63} ERROR - Error executing query: (1062, "Duplicate entry '2025-03-30 11:17:01-9176' for key 'PRIMARY'")
[2025-03-30T11:17:01.325+0000] {inout_data.py:63} ERROR - Error executing query: (1062, "Duplicate entry '2025-03-30 11:17:01-9176' for key 'PRIMARY'")
[2025-03-30T11:17:01.325+0000] {inout_data.py:63} ERROR - Error executing query: (1062, "Duplicate entry '2025-03-30 11:17:01-9176' for key 'PRIMARY'")
[2025-03-30T11:17:01.325+0000] {inout_data.py:63} ERROR - Error executing query: (1062, "Duplicate entry '2025-03-30 11:17:01-9176' for key 'PRIMARY'")
[2025-03-30T11:17:01.325+0000] {inout_data.py:63} ERROR - Error executing query: (1062, "Duplicate entry '2025-03-30 11:17:01-9176' for key 'PRIMARY'")
[2025-03-30T11:17:01.326+0000] {inout_data.py:63} ERROR - Error executing query: (1062, "Duplicate entry '2025-03-30 11:17:01-9176' for key 'PRIMARY'")
[2025-03-30T11:17:01.326+0000] {inout_data.py:63} ERROR - Error executing query: (1062, "Duplicate entry '2025-03-30 11:17:01-9176' for key 'PRIMARY'")
[2025-03-30T11:17:01.326+0000] {inout_data.py:63} ERROR - Error executing query: (1062, "Duplicate entry '2025-03-30 11:17:01-9176' for key 'PRIMARY'")
[2025-03-30T11:17:01.327+0000] {inout_data.py:63} ERROR - Error executing query: (1062, "Duplicate entry '2025-03-30 11:17:01-9176' for key 'PRIMARY'")
[2025-03-30T11:17:01.327+0000] {inout_data.py:152} INFO - Ingested 13 rows of dummy data.
[2025-03-30T11:17:01.327+0000] {inout_data.py:78} INFO - MariaDB connection closed.
[2025-03-30T11:17:01.327+0000] {python.py:202} INFO - Done. Returned value was: None
[2025-03-30T11:17:01.333+0000] {taskinstance.py:1149} INFO - Marking task as SUCCESS. dag_id=inout_data_ingestion, task_id=ingest_dummy_data, execution_date=20241002T013000, start_date=20250330T111701, end_date=20250330T111701
[2025-03-30T11:17:01.380+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2025-03-30T11:17:01.393+0000] {taskinstance.py:3309} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-03-30T11:29:31.690+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: inout_data_ingestion.ingest_dummy_data scheduled__2024-10-02T01:30:00+00:00 [queued]>
[2025-03-30T11:29:31.696+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: inout_data_ingestion.ingest_dummy_data scheduled__2024-10-02T01:30:00+00:00 [queued]>
[2025-03-30T11:29:31.696+0000] {taskinstance.py:2193} INFO - Starting attempt 1 of 2
[2025-03-30T11:29:31.704+0000] {taskinstance.py:2214} INFO - Executing <Task(PythonOperator): ingest_dummy_data> on 2024-10-02 01:30:00+00:00
[2025-03-30T11:29:31.707+0000] {standard_task_runner.py:60} INFO - Started process 4466 to run task
[2025-03-30T11:29:31.710+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'inout_data_ingestion', 'ingest_dummy_data', 'scheduled__2024-10-02T01:30:00+00:00', '--job-id', '2208', '--raw', '--subdir', 'DAGS_FOLDER/inout_data.py', '--cfg-path', '/tmp/tmpvs4ddsh7']
[2025-03-30T11:29:31.711+0000] {standard_task_runner.py:88} INFO - Job 2208: Subtask ingest_dummy_data
[2025-03-30T11:29:31.718+0000] {warnings.py:109} WARNING - /home/***/.local/lib/python3.8/site-packages/***/settings.py:194: DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
  SQL_ALCHEMY_CONN = conf.get("database", "SQL_ALCHEMY_CONN")

[2025-03-30T11:29:31.735+0000] {task_command.py:423} INFO - Running <TaskInstance: inout_data_ingestion.ingest_dummy_data scheduled__2024-10-02T01:30:00+00:00 [running]> on host ff9d64b065e6
[2025-03-30T11:29:31.782+0000] {taskinstance.py:2510} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='inout_data_ingestion' AIRFLOW_CTX_TASK_ID='ingest_dummy_data' AIRFLOW_CTX_EXECUTION_DATE='2024-10-02T01:30:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-10-02T01:30:00+00:00'
[2025-03-30T11:29:31.788+0000] {logging_mixin.py:188} INFO - Successfully connected to MariaDB: INOUT on mariadb:3306
[2025-03-30T11:29:31.789+0000] {logging_mixin.py:188} INFO - General error executing query: not all arguments converted during string formatting
[2025-03-30T11:29:31.789+0000] {logging_mixin.py:188} INFO - General error executing query: not all arguments converted during string formatting
[2025-03-30T11:29:31.789+0000] {logging_mixin.py:188} INFO - General error executing query: not all arguments converted during string formatting
[2025-03-30T11:29:31.790+0000] {logging_mixin.py:188} INFO - General error executing query: not all arguments converted during string formatting
[2025-03-30T11:29:31.790+0000] {logging_mixin.py:188} INFO - General error executing query: not all arguments converted during string formatting
[2025-03-30T11:29:31.790+0000] {logging_mixin.py:188} INFO - General error executing query: not all arguments converted during string formatting
[2025-03-30T11:29:31.790+0000] {logging_mixin.py:188} INFO - General error executing query: not all arguments converted during string formatting
[2025-03-30T11:29:31.790+0000] {logging_mixin.py:188} INFO - General error executing query: not all arguments converted during string formatting
[2025-03-30T11:29:31.791+0000] {logging_mixin.py:188} INFO - General error executing query: not all arguments converted during string formatting
[2025-03-30T11:29:31.791+0000] {logging_mixin.py:188} INFO - General error executing query: not all arguments converted during string formatting
[2025-03-30T11:29:31.791+0000] {logging_mixin.py:188} INFO - General error executing query: not all arguments converted during string formatting
[2025-03-30T11:29:31.791+0000] {logging_mixin.py:188} INFO - General error executing query: not all arguments converted during string formatting
[2025-03-30T11:29:31.792+0000] {inout_data.py:87} INFO - Ingested 12 rows of dummy data for 2024-10-02 01:30:00+00:00.
[2025-03-30T11:29:31.792+0000] {logging_mixin.py:188} INFO - MariaDB connection closed.
[2025-03-30T11:29:31.792+0000] {python.py:202} INFO - Done. Returned value was: None
[2025-03-30T11:29:31.796+0000] {taskinstance.py:1149} INFO - Marking task as SUCCESS. dag_id=inout_data_ingestion, task_id=ingest_dummy_data, execution_date=20241002T013000, start_date=20250330T112931, end_date=20250330T112931
[2025-03-30T11:29:31.807+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2025-03-30T11:29:31.815+0000] {taskinstance.py:3309} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-03-30T11:34:22.676+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: inout_data_ingestion.ingest_dummy_data scheduled__2024-10-02T01:30:00+00:00 [queued]>
[2025-03-30T11:34:22.685+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: inout_data_ingestion.ingest_dummy_data scheduled__2024-10-02T01:30:00+00:00 [queued]>
[2025-03-30T11:34:22.685+0000] {taskinstance.py:2193} INFO - Starting attempt 1 of 2
[2025-03-30T11:34:22.703+0000] {taskinstance.py:2214} INFO - Executing <Task(PythonOperator): ingest_dummy_data> on 2024-10-02 01:30:00+00:00
[2025-03-30T11:34:22.707+0000] {standard_task_runner.py:60} INFO - Started process 5923 to run task
[2025-03-30T11:34:22.709+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'inout_data_ingestion', 'ingest_dummy_data', 'scheduled__2024-10-02T01:30:00+00:00', '--job-id', '2936', '--raw', '--subdir', 'DAGS_FOLDER/inout_data.py', '--cfg-path', '/tmp/tmpz_fvp_ua']
[2025-03-30T11:34:22.711+0000] {standard_task_runner.py:88} INFO - Job 2936: Subtask ingest_dummy_data
[2025-03-30T11:34:22.722+0000] {warnings.py:109} WARNING - /home/***/.local/lib/python3.8/site-packages/***/settings.py:194: DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
  SQL_ALCHEMY_CONN = conf.get("database", "SQL_ALCHEMY_CONN")

[2025-03-30T11:34:22.750+0000] {task_command.py:423} INFO - Running <TaskInstance: inout_data_ingestion.ingest_dummy_data scheduled__2024-10-02T01:30:00+00:00 [running]> on host ff9d64b065e6
[2025-03-30T11:34:22.804+0000] {taskinstance.py:2510} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='inout_data_ingestion' AIRFLOW_CTX_TASK_ID='ingest_dummy_data' AIRFLOW_CTX_EXECUTION_DATE='2024-10-02T01:30:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-10-02T01:30:00+00:00'
[2025-03-30T11:34:22.810+0000] {logging_mixin.py:188} INFO - Successfully connected to MariaDB: INOUT on mariadb:3306
[2025-03-30T11:34:22.810+0000] {inout_data.py:83} INFO - {'ingested_time': datetime.datetime(2025, 3, 30, 11, 34, 22, 810574), 'timestamp': datetime.datetime(2024, 10, 2, 1, 45, tzinfo=datetime.timezone.utc), 'emp_id': 9773, 'employee_name': 'Employee10', 'factory': 'FactoryA', 'in_out': 'IN', 'entrance': 'Side', 'door_number': '8', 'function': 'IT', 'division': 'AAPD', 'department': 'ML', 'section': '1'}
[2025-03-30T11:34:22.811+0000] {logging_mixin.py:188} INFO - General error executing query: not all arguments converted during string formatting
[2025-03-30T11:34:22.811+0000] {inout_data.py:83} INFO - {'ingested_time': datetime.datetime(2025, 3, 30, 11, 34, 22, 811254), 'timestamp': datetime.datetime(2024, 10, 2, 1, 50, tzinfo=datetime.timezone.utc), 'emp_id': 1038, 'employee_name': 'Employee34', 'factory': 'FactoryB', 'in_out': 'OUT', 'entrance': 'Side', 'door_number': '8', 'function': 'IT', 'division': 'AAPD', 'department': 'CAPD', 'section': '1'}
[2025-03-30T11:34:22.811+0000] {logging_mixin.py:188} INFO - General error executing query: not all arguments converted during string formatting
[2025-03-30T11:34:22.811+0000] {inout_data.py:83} INFO - {'ingested_time': datetime.datetime(2025, 3, 30, 11, 34, 22, 811792), 'timestamp': datetime.datetime(2024, 10, 2, 1, 31, tzinfo=datetime.timezone.utc), 'emp_id': 5458, 'employee_name': 'Employee69', 'factory': 'FactoryA', 'in_out': 'IN', 'entrance': 'Side', 'door_number': '9', 'function': 'IT', 'division': 'AAPD', 'department': 'CAPD', 'section': '2'}
[2025-03-30T11:34:22.812+0000] {logging_mixin.py:188} INFO - General error executing query: not all arguments converted during string formatting
[2025-03-30T11:34:22.812+0000] {inout_data.py:83} INFO - {'ingested_time': datetime.datetime(2025, 3, 30, 11, 34, 22, 812516), 'timestamp': datetime.datetime(2024, 10, 2, 1, 54, tzinfo=datetime.timezone.utc), 'emp_id': 5952, 'employee_name': 'Employee88', 'factory': 'FactoryA', 'in_out': 'IN', 'entrance': 'Main', 'door_number': '1', 'function': 'IT', 'division': 'AAPD', 'department': 'CAPD', 'section': '2'}
[2025-03-30T11:34:22.812+0000] {logging_mixin.py:188} INFO - General error executing query: not all arguments converted during string formatting
[2025-03-30T11:34:22.813+0000] {inout_data.py:83} INFO - {'ingested_time': datetime.datetime(2025, 3, 30, 11, 34, 22, 813190), 'timestamp': datetime.datetime(2024, 10, 2, 1, 39, tzinfo=datetime.timezone.utc), 'emp_id': 5725, 'employee_name': 'Employee95', 'factory': 'FactoryB', 'in_out': 'IN', 'entrance': 'Main', 'door_number': '1', 'function': 'IT', 'division': 'AAPD', 'department': 'CAPD', 'section': '1'}
[2025-03-30T11:34:22.813+0000] {logging_mixin.py:188} INFO - General error executing query: not all arguments converted during string formatting
[2025-03-30T11:34:22.813+0000] {inout_data.py:83} INFO - {'ingested_time': datetime.datetime(2025, 3, 30, 11, 34, 22, 813734), 'timestamp': datetime.datetime(2024, 10, 2, 1, 34, tzinfo=datetime.timezone.utc), 'emp_id': 7452, 'employee_name': 'Employee72', 'factory': 'FactoryA', 'in_out': 'OUT', 'entrance': 'Main', 'door_number': '1', 'function': 'IT', 'division': 'AAPD', 'department': 'CAPD', 'section': '2'}
[2025-03-30T11:34:22.813+0000] {logging_mixin.py:188} INFO - General error executing query: not all arguments converted during string formatting
[2025-03-30T11:34:22.814+0000] {inout_data.py:83} INFO - {'ingested_time': datetime.datetime(2025, 3, 30, 11, 34, 22, 814089), 'timestamp': datetime.datetime(2024, 10, 2, 1, 55, tzinfo=datetime.timezone.utc), 'emp_id': 8071, 'employee_name': 'Employee34', 'factory': 'FactoryA', 'in_out': 'OUT', 'entrance': 'Main', 'door_number': '1', 'function': 'IT', 'division': 'AAPD', 'department': 'HRSD', 'section': '2'}
[2025-03-30T11:34:22.814+0000] {logging_mixin.py:188} INFO - General error executing query: not all arguments converted during string formatting
[2025-03-30T11:34:22.814+0000] {inout_data.py:83} INFO - {'ingested_time': datetime.datetime(2025, 3, 30, 11, 34, 22, 814526), 'timestamp': datetime.datetime(2024, 10, 2, 1, 57, tzinfo=datetime.timezone.utc), 'emp_id': 3930, 'employee_name': 'Employee30', 'factory': 'FactoryB', 'in_out': 'OUT', 'entrance': 'Side', 'door_number': '1', 'function': 'IT', 'division': 'AAPD', 'department': 'ML', 'section': '1'}
[2025-03-30T11:34:22.814+0000] {logging_mixin.py:188} INFO - General error executing query: not all arguments converted during string formatting
[2025-03-30T11:34:22.814+0000] {inout_data.py:88} INFO - Ingested 8 rows of dummy data for 2024-10-02 01:30:00+00:00.
[2025-03-30T11:34:22.815+0000] {logging_mixin.py:188} INFO - MariaDB connection closed.
[2025-03-30T11:34:22.815+0000] {python.py:202} INFO - Done. Returned value was: None
[2025-03-30T11:34:22.820+0000] {taskinstance.py:1149} INFO - Marking task as SUCCESS. dag_id=inout_data_ingestion, task_id=ingest_dummy_data, execution_date=20241002T013000, start_date=20250330T113422, end_date=20250330T113422
[2025-03-30T11:34:22.848+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2025-03-30T11:34:22.859+0000] {taskinstance.py:3309} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-03-30T11:41:59.934+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: inout_data_ingestion.ingest_dummy_data scheduled__2024-10-02T01:30:00+00:00 [queued]>
[2025-03-30T11:41:59.947+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: inout_data_ingestion.ingest_dummy_data scheduled__2024-10-02T01:30:00+00:00 [queued]>
[2025-03-30T11:41:59.948+0000] {taskinstance.py:2193} INFO - Starting attempt 1 of 2
[2025-03-30T11:41:59.976+0000] {taskinstance.py:2214} INFO - Executing <Task(PythonOperator): ingest_dummy_data> on 2024-10-02 01:30:00+00:00
[2025-03-30T11:41:59.987+0000] {standard_task_runner.py:60} INFO - Started process 7271 to run task
[2025-03-30T11:42:00.015+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'inout_data_ingestion', 'ingest_dummy_data', 'scheduled__2024-10-02T01:30:00+00:00', '--job-id', '3610', '--raw', '--subdir', 'DAGS_FOLDER/inout_data.py', '--cfg-path', '/tmp/tmp56qt5_p5']
[2025-03-30T11:42:00.021+0000] {standard_task_runner.py:88} INFO - Job 3610: Subtask ingest_dummy_data
[2025-03-30T11:42:00.049+0000] {warnings.py:109} WARNING - /home/***/.local/lib/python3.8/site-packages/***/settings.py:194: DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
  SQL_ALCHEMY_CONN = conf.get("database", "SQL_ALCHEMY_CONN")

[2025-03-30T11:42:00.089+0000] {task_command.py:423} INFO - Running <TaskInstance: inout_data_ingestion.ingest_dummy_data scheduled__2024-10-02T01:30:00+00:00 [running]> on host ff9d64b065e6
[2025-03-30T11:42:00.172+0000] {taskinstance.py:2510} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='inout_data_ingestion' AIRFLOW_CTX_TASK_ID='ingest_dummy_data' AIRFLOW_CTX_EXECUTION_DATE='2024-10-02T01:30:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-10-02T01:30:00+00:00'
[2025-03-30T11:42:00.178+0000] {logging_mixin.py:188} INFO - Successfully connected to MariaDB: INOUT on mariadb:3306
[2025-03-30T11:42:00.179+0000] {inout_data.py:83} INFO - {'ingested_time': datetime.datetime(2025, 3, 30, 11, 42, 0, 179213), 'timestamp': datetime.datetime(2024, 10, 2, 1, 54, tzinfo=datetime.timezone.utc), 'emp_id': 6725, 'employee_name': 'Employee40', 'factory': 'FactoryA', 'in_out': 'IN', 'entrance': 'Main', 'door_number': '8', 'function': 'IT', 'division': 'BSID', 'department': 'HRSD', 'section': '2'}
[2025-03-30T11:42:00.179+0000] {inout_data.py:99} INFO - Query: INSERT INTO emp_inout_rec (ingested_time, timestamp, employee_id, employee_name, factory, in_out, entrance, door_number, function, division, department, section) VALUES (%s, %s, %s, %s, %s, %s, %s, %s, %s, %s, %s)
[2025-03-30T11:42:00.180+0000] {inout_data.py:100} INFO - Params: ('2025-03-30 11:42:00', '2024-10-02 01:54:00', 6725, 'Employee40', 'FactoryA', 'IN', 'Main', '8', 'IT', 'BSID', 'HRSD', '2')
[2025-03-30T11:42:00.180+0000] {logging_mixin.py:188} INFO - General error executing query: not all arguments converted during string formatting
[2025-03-30T11:42:00.180+0000] {inout_data.py:83} INFO - {'ingested_time': datetime.datetime(2025, 3, 30, 11, 42, 0, 180722), 'timestamp': datetime.datetime(2024, 10, 2, 1, 44, tzinfo=datetime.timezone.utc), 'emp_id': 9128, 'employee_name': 'Employee76', 'factory': 'FactoryB', 'in_out': 'OUT', 'entrance': 'Main', 'door_number': '4', 'function': 'IT', 'division': 'AAPD', 'department': 'ML', 'section': '1'}
[2025-03-30T11:42:00.181+0000] {inout_data.py:99} INFO - Query: INSERT INTO emp_inout_rec (ingested_time, timestamp, employee_id, employee_name, factory, in_out, entrance, door_number, function, division, department, section) VALUES (%s, %s, %s, %s, %s, %s, %s, %s, %s, %s, %s)
[2025-03-30T11:42:00.181+0000] {inout_data.py:100} INFO - Params: ('2025-03-30 11:42:00', '2024-10-02 01:44:00', 9128, 'Employee76', 'FactoryB', 'OUT', 'Main', '4', 'IT', 'AAPD', 'ML', '1')
[2025-03-30T11:42:00.182+0000] {logging_mixin.py:188} INFO - General error executing query: not all arguments converted during string formatting
[2025-03-30T11:42:00.183+0000] {inout_data.py:83} INFO - {'ingested_time': datetime.datetime(2025, 3, 30, 11, 42, 0, 183247), 'timestamp': datetime.datetime(2024, 10, 2, 1, 30, tzinfo=datetime.timezone.utc), 'emp_id': 1445, 'employee_name': 'Employee38', 'factory': 'FactoryB', 'in_out': 'OUT', 'entrance': 'Side', 'door_number': '4', 'function': 'IT', 'division': 'BSID', 'department': 'CAPD', 'section': '2'}
[2025-03-30T11:42:00.183+0000] {inout_data.py:99} INFO - Query: INSERT INTO emp_inout_rec (ingested_time, timestamp, employee_id, employee_name, factory, in_out, entrance, door_number, function, division, department, section) VALUES (%s, %s, %s, %s, %s, %s, %s, %s, %s, %s, %s)
[2025-03-30T11:42:00.184+0000] {inout_data.py:100} INFO - Params: ('2025-03-30 11:42:00', '2024-10-02 01:30:00', 1445, 'Employee38', 'FactoryB', 'OUT', 'Side', '4', 'IT', 'BSID', 'CAPD', '2')
[2025-03-30T11:42:00.184+0000] {logging_mixin.py:188} INFO - General error executing query: not all arguments converted during string formatting
[2025-03-30T11:42:00.185+0000] {inout_data.py:83} INFO - {'ingested_time': datetime.datetime(2025, 3, 30, 11, 42, 0, 185045), 'timestamp': datetime.datetime(2024, 10, 2, 1, 47, tzinfo=datetime.timezone.utc), 'emp_id': 6723, 'employee_name': 'Employee48', 'factory': 'FactoryB', 'in_out': 'OUT', 'entrance': 'Side', 'door_number': '6', 'function': 'IT', 'division': 'BSID', 'department': 'CAPD', 'section': '1'}
[2025-03-30T11:42:00.185+0000] {inout_data.py:99} INFO - Query: INSERT INTO emp_inout_rec (ingested_time, timestamp, employee_id, employee_name, factory, in_out, entrance, door_number, function, division, department, section) VALUES (%s, %s, %s, %s, %s, %s, %s, %s, %s, %s, %s)
[2025-03-30T11:42:00.185+0000] {inout_data.py:100} INFO - Params: ('2025-03-30 11:42:00', '2024-10-02 01:47:00', 6723, 'Employee48', 'FactoryB', 'OUT', 'Side', '6', 'IT', 'BSID', 'CAPD', '1')
[2025-03-30T11:42:00.186+0000] {logging_mixin.py:188} INFO - General error executing query: not all arguments converted during string formatting
[2025-03-30T11:42:00.186+0000] {inout_data.py:83} INFO - {'ingested_time': datetime.datetime(2025, 3, 30, 11, 42, 0, 186909), 'timestamp': datetime.datetime(2024, 10, 2, 1, 39, tzinfo=datetime.timezone.utc), 'emp_id': 8400, 'employee_name': 'Employee21', 'factory': 'FactoryB', 'in_out': 'OUT', 'entrance': 'Main', 'door_number': '4', 'function': 'IT', 'division': 'AAPD', 'department': 'CAPD', 'section': '1'}
[2025-03-30T11:42:00.187+0000] {inout_data.py:99} INFO - Query: INSERT INTO emp_inout_rec (ingested_time, timestamp, employee_id, employee_name, factory, in_out, entrance, door_number, function, division, department, section) VALUES (%s, %s, %s, %s, %s, %s, %s, %s, %s, %s, %s)
[2025-03-30T11:42:00.187+0000] {inout_data.py:100} INFO - Params: ('2025-03-30 11:42:00', '2024-10-02 01:39:00', 8400, 'Employee21', 'FactoryB', 'OUT', 'Main', '4', 'IT', 'AAPD', 'CAPD', '1')
[2025-03-30T11:42:00.188+0000] {logging_mixin.py:188} INFO - General error executing query: not all arguments converted during string formatting
[2025-03-30T11:42:00.189+0000] {inout_data.py:83} INFO - {'ingested_time': datetime.datetime(2025, 3, 30, 11, 42, 0, 189096), 'timestamp': datetime.datetime(2024, 10, 2, 1, 55, tzinfo=datetime.timezone.utc), 'emp_id': 7821, 'employee_name': 'Employee1', 'factory': 'FactoryA', 'in_out': 'OUT', 'entrance': 'Main', 'door_number': '10', 'function': 'IT', 'division': 'AAPD', 'department': 'ML', 'section': '1'}
[2025-03-30T11:42:00.189+0000] {inout_data.py:99} INFO - Query: INSERT INTO emp_inout_rec (ingested_time, timestamp, employee_id, employee_name, factory, in_out, entrance, door_number, function, division, department, section) VALUES (%s, %s, %s, %s, %s, %s, %s, %s, %s, %s, %s)
[2025-03-30T11:42:00.190+0000] {inout_data.py:100} INFO - Params: ('2025-03-30 11:42:00', '2024-10-02 01:55:00', 7821, 'Employee1', 'FactoryA', 'OUT', 'Main', '10', 'IT', 'AAPD', 'ML', '1')
[2025-03-30T11:42:00.191+0000] {logging_mixin.py:188} INFO - General error executing query: not all arguments converted during string formatting
[2025-03-30T11:42:00.192+0000] {inout_data.py:83} INFO - {'ingested_time': datetime.datetime(2025, 3, 30, 11, 42, 0, 192191), 'timestamp': datetime.datetime(2024, 10, 2, 1, 41, tzinfo=datetime.timezone.utc), 'emp_id': 5429, 'employee_name': 'Employee92', 'factory': 'FactoryB', 'in_out': 'OUT', 'entrance': 'Main', 'door_number': '4', 'function': 'IT', 'division': 'BSID', 'department': 'CAPD', 'section': '2'}
[2025-03-30T11:42:00.193+0000] {inout_data.py:99} INFO - Query: INSERT INTO emp_inout_rec (ingested_time, timestamp, employee_id, employee_name, factory, in_out, entrance, door_number, function, division, department, section) VALUES (%s, %s, %s, %s, %s, %s, %s, %s, %s, %s, %s)
[2025-03-30T11:42:00.193+0000] {inout_data.py:100} INFO - Params: ('2025-03-30 11:42:00', '2024-10-02 01:41:00', 5429, 'Employee92', 'FactoryB', 'OUT', 'Main', '4', 'IT', 'BSID', 'CAPD', '2')
[2025-03-30T11:42:00.194+0000] {logging_mixin.py:188} INFO - General error executing query: not all arguments converted during string formatting
[2025-03-30T11:42:00.195+0000] {inout_data.py:83} INFO - {'ingested_time': datetime.datetime(2025, 3, 30, 11, 42, 0, 195147), 'timestamp': datetime.datetime(2024, 10, 2, 1, 47, tzinfo=datetime.timezone.utc), 'emp_id': 1530, 'employee_name': 'Employee95', 'factory': 'FactoryB', 'in_out': 'IN', 'entrance': 'Main', 'door_number': '7', 'function': 'IT', 'division': 'AAPD', 'department': 'HRSD', 'section': '1'}
[2025-03-30T11:42:00.196+0000] {inout_data.py:99} INFO - Query: INSERT INTO emp_inout_rec (ingested_time, timestamp, employee_id, employee_name, factory, in_out, entrance, door_number, function, division, department, section) VALUES (%s, %s, %s, %s, %s, %s, %s, %s, %s, %s, %s)
[2025-03-30T11:42:00.196+0000] {inout_data.py:100} INFO - Params: ('2025-03-30 11:42:00', '2024-10-02 01:47:00', 1530, 'Employee95', 'FactoryB', 'IN', 'Main', '7', 'IT', 'AAPD', 'HRSD', '1')
[2025-03-30T11:42:00.197+0000] {logging_mixin.py:188} INFO - General error executing query: not all arguments converted during string formatting
[2025-03-30T11:42:00.198+0000] {inout_data.py:83} INFO - {'ingested_time': datetime.datetime(2025, 3, 30, 11, 42, 0, 198006), 'timestamp': datetime.datetime(2024, 10, 2, 1, 31, tzinfo=datetime.timezone.utc), 'emp_id': 4932, 'employee_name': 'Employee21', 'factory': 'FactoryA', 'in_out': 'IN', 'entrance': 'Side', 'door_number': '10', 'function': 'IT', 'division': 'AAPD', 'department': 'CAPD', 'section': '1'}
[2025-03-30T11:42:00.198+0000] {inout_data.py:99} INFO - Query: INSERT INTO emp_inout_rec (ingested_time, timestamp, employee_id, employee_name, factory, in_out, entrance, door_number, function, division, department, section) VALUES (%s, %s, %s, %s, %s, %s, %s, %s, %s, %s, %s)
[2025-03-30T11:42:00.199+0000] {inout_data.py:100} INFO - Params: ('2025-03-30 11:42:00', '2024-10-02 01:31:00', 4932, 'Employee21', 'FactoryA', 'IN', 'Side', '10', 'IT', 'AAPD', 'CAPD', '1')
[2025-03-30T11:42:00.200+0000] {logging_mixin.py:188} INFO - General error executing query: not all arguments converted during string formatting
[2025-03-30T11:42:00.202+0000] {inout_data.py:83} INFO - {'ingested_time': datetime.datetime(2025, 3, 30, 11, 42, 0, 202272), 'timestamp': datetime.datetime(2024, 10, 2, 1, 32, tzinfo=datetime.timezone.utc), 'emp_id': 2197, 'employee_name': 'Employee94', 'factory': 'FactoryA', 'in_out': 'OUT', 'entrance': 'Main', 'door_number': '7', 'function': 'IT', 'division': 'AAPD', 'department': 'HRSD', 'section': '1'}
[2025-03-30T11:42:00.203+0000] {inout_data.py:99} INFO - Query: INSERT INTO emp_inout_rec (ingested_time, timestamp, employee_id, employee_name, factory, in_out, entrance, door_number, function, division, department, section) VALUES (%s, %s, %s, %s, %s, %s, %s, %s, %s, %s, %s)
[2025-03-30T11:42:00.204+0000] {inout_data.py:100} INFO - Params: ('2025-03-30 11:42:00', '2024-10-02 01:32:00', 2197, 'Employee94', 'FactoryA', 'OUT', 'Main', '7', 'IT', 'AAPD', 'HRSD', '1')
[2025-03-30T11:42:00.205+0000] {logging_mixin.py:188} INFO - General error executing query: not all arguments converted during string formatting
[2025-03-30T11:42:00.206+0000] {inout_data.py:83} INFO - {'ingested_time': datetime.datetime(2025, 3, 30, 11, 42, 0, 206219), 'timestamp': datetime.datetime(2024, 10, 2, 1, 34, tzinfo=datetime.timezone.utc), 'emp_id': 4883, 'employee_name': 'Employee14', 'factory': 'FactoryA', 'in_out': 'OUT', 'entrance': 'Main', 'door_number': '1', 'function': 'IT', 'division': 'BSID', 'department': 'CAPD', 'section': '2'}
[2025-03-30T11:42:00.207+0000] {inout_data.py:99} INFO - Query: INSERT INTO emp_inout_rec (ingested_time, timestamp, employee_id, employee_name, factory, in_out, entrance, door_number, function, division, department, section) VALUES (%s, %s, %s, %s, %s, %s, %s, %s, %s, %s, %s)
[2025-03-30T11:42:00.207+0000] {inout_data.py:100} INFO - Params: ('2025-03-30 11:42:00', '2024-10-02 01:34:00', 4883, 'Employee14', 'FactoryA', 'OUT', 'Main', '1', 'IT', 'BSID', 'CAPD', '2')
[2025-03-30T11:42:00.208+0000] {logging_mixin.py:188} INFO - General error executing query: not all arguments converted during string formatting
[2025-03-30T11:42:00.209+0000] {inout_data.py:83} INFO - {'ingested_time': datetime.datetime(2025, 3, 30, 11, 42, 0, 209464), 'timestamp': datetime.datetime(2024, 10, 2, 1, 51, tzinfo=datetime.timezone.utc), 'emp_id': 1683, 'employee_name': 'Employee24', 'factory': 'FactoryA', 'in_out': 'IN', 'entrance': 'Side', 'door_number': '4', 'function': 'IT', 'division': 'AAPD', 'department': 'HRSD', 'section': '2'}
[2025-03-30T11:42:00.209+0000] {inout_data.py:99} INFO - Query: INSERT INTO emp_inout_rec (ingested_time, timestamp, employee_id, employee_name, factory, in_out, entrance, door_number, function, division, department, section) VALUES (%s, %s, %s, %s, %s, %s, %s, %s, %s, %s, %s)
[2025-03-30T11:42:00.210+0000] {inout_data.py:100} INFO - Params: ('2025-03-30 11:42:00', '2024-10-02 01:51:00', 1683, 'Employee24', 'FactoryA', 'IN', 'Side', '4', 'IT', 'AAPD', 'HRSD', '2')
[2025-03-30T11:42:00.210+0000] {logging_mixin.py:188} INFO - General error executing query: not all arguments converted during string formatting
[2025-03-30T11:42:00.211+0000] {inout_data.py:83} INFO - {'ingested_time': datetime.datetime(2025, 3, 30, 11, 42, 0, 211286), 'timestamp': datetime.datetime(2024, 10, 2, 1, 34, tzinfo=datetime.timezone.utc), 'emp_id': 1204, 'employee_name': 'Employee33', 'factory': 'FactoryA', 'in_out': 'OUT', 'entrance': 'Main', 'door_number': '5', 'function': 'IT', 'division': 'AAPD', 'department': 'CAPD', 'section': '1'}
[2025-03-30T11:42:00.212+0000] {inout_data.py:99} INFO - Query: INSERT INTO emp_inout_rec (ingested_time, timestamp, employee_id, employee_name, factory, in_out, entrance, door_number, function, division, department, section) VALUES (%s, %s, %s, %s, %s, %s, %s, %s, %s, %s, %s)
[2025-03-30T11:42:00.212+0000] {inout_data.py:100} INFO - Params: ('2025-03-30 11:42:00', '2024-10-02 01:34:00', 1204, 'Employee33', 'FactoryA', 'OUT', 'Main', '5', 'IT', 'AAPD', 'CAPD', '1')
[2025-03-30T11:42:00.213+0000] {logging_mixin.py:188} INFO - General error executing query: not all arguments converted during string formatting
[2025-03-30T11:42:00.213+0000] {inout_data.py:83} INFO - {'ingested_time': datetime.datetime(2025, 3, 30, 11, 42, 0, 213917), 'timestamp': datetime.datetime(2024, 10, 2, 1, 30, tzinfo=datetime.timezone.utc), 'emp_id': 4360, 'employee_name': 'Employee58', 'factory': 'FactoryA', 'in_out': 'OUT', 'entrance': 'Side', 'door_number': '6', 'function': 'IT', 'division': 'BSID', 'department': 'ML', 'section': '2'}
[2025-03-30T11:42:00.214+0000] {inout_data.py:99} INFO - Query: INSERT INTO emp_inout_rec (ingested_time, timestamp, employee_id, employee_name, factory, in_out, entrance, door_number, function, division, department, section) VALUES (%s, %s, %s, %s, %s, %s, %s, %s, %s, %s, %s)
[2025-03-30T11:42:00.215+0000] {inout_data.py:100} INFO - Params: ('2025-03-30 11:42:00', '2024-10-02 01:30:00', 4360, 'Employee58', 'FactoryA', 'OUT', 'Side', '6', 'IT', 'BSID', 'ML', '2')
[2025-03-30T11:42:00.216+0000] {logging_mixin.py:188} INFO - General error executing query: not all arguments converted during string formatting
[2025-03-30T11:42:00.218+0000] {inout_data.py:83} INFO - {'ingested_time': datetime.datetime(2025, 3, 30, 11, 42, 0, 218155), 'timestamp': datetime.datetime(2024, 10, 2, 1, 37, tzinfo=datetime.timezone.utc), 'emp_id': 1566, 'employee_name': 'Employee23', 'factory': 'FactoryA', 'in_out': 'IN', 'entrance': 'Main', 'door_number': '5', 'function': 'IT', 'division': 'BSID', 'department': 'HRSD', 'section': '1'}
[2025-03-30T11:42:00.219+0000] {inout_data.py:99} INFO - Query: INSERT INTO emp_inout_rec (ingested_time, timestamp, employee_id, employee_name, factory, in_out, entrance, door_number, function, division, department, section) VALUES (%s, %s, %s, %s, %s, %s, %s, %s, %s, %s, %s)
[2025-03-30T11:42:00.219+0000] {inout_data.py:100} INFO - Params: ('2025-03-30 11:42:00', '2024-10-02 01:37:00', 1566, 'Employee23', 'FactoryA', 'IN', 'Main', '5', 'IT', 'BSID', 'HRSD', '1')
[2025-03-30T11:42:00.220+0000] {logging_mixin.py:188} INFO - General error executing query: not all arguments converted during string formatting
[2025-03-30T11:42:00.221+0000] {inout_data.py:83} INFO - {'ingested_time': datetime.datetime(2025, 3, 30, 11, 42, 0, 221398), 'timestamp': datetime.datetime(2024, 10, 2, 1, 45, tzinfo=datetime.timezone.utc), 'emp_id': 3178, 'employee_name': 'Employee92', 'factory': 'FactoryB', 'in_out': 'IN', 'entrance': 'Main', 'door_number': '9', 'function': 'IT', 'division': 'AAPD', 'department': 'ML', 'section': '1'}
[2025-03-30T11:42:00.222+0000] {inout_data.py:99} INFO - Query: INSERT INTO emp_inout_rec (ingested_time, timestamp, employee_id, employee_name, factory, in_out, entrance, door_number, function, division, department, section) VALUES (%s, %s, %s, %s, %s, %s, %s, %s, %s, %s, %s)
[2025-03-30T11:42:00.223+0000] {inout_data.py:100} INFO - Params: ('2025-03-30 11:42:00', '2024-10-02 01:45:00', 3178, 'Employee92', 'FactoryB', 'IN', 'Main', '9', 'IT', 'AAPD', 'ML', '1')
[2025-03-30T11:42:00.223+0000] {logging_mixin.py:188} INFO - General error executing query: not all arguments converted during string formatting
[2025-03-30T11:42:00.224+0000] {inout_data.py:102} INFO - Ingested 16 rows of dummy data for 2024-10-02 01:30:00+00:00.
[2025-03-30T11:42:00.224+0000] {logging_mixin.py:188} INFO - MariaDB connection closed.
[2025-03-30T11:42:00.225+0000] {python.py:202} INFO - Done. Returned value was: None
[2025-03-30T11:42:00.234+0000] {taskinstance.py:1149} INFO - Marking task as SUCCESS. dag_id=inout_data_ingestion, task_id=ingest_dummy_data, execution_date=20241002T013000, start_date=20250330T114159, end_date=20250330T114200
[2025-03-30T11:42:00.267+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2025-03-30T11:42:00.285+0000] {taskinstance.py:3309} INFO - 0 downstream tasks scheduled from follow-on schedule check
